### PR TITLE
fix(management): addition of createAt, updatedAl & lastConnectionAt i…

### PR DESCRIPTION
**Pull request by leleueri**
…n bean provided by /management/user endpoint

fix gravitee-io/issues#4300

### CODE REVIEW

File: gravitee-management-api-idp/gravitee-management-api-idp-api/src/main/java/io/gravitee/management/idp/api/authentication/UserDetails.java [hunk 1]
```diff
@@ -39,7 +40,20 @@
     private List<UserDetailRole> roles;
     private String username;
     private byte[] picture;
+    /**
+     * The user creation date
+     */
+    private Date createdAt;
```
Comment (brasseld) [inline gravitee-management-api-idp/gravitee-management-api-idp-api/src/main/java/io/gravitee/management/idp/api/authentication/UserDetails.java:46]: Please force the property name to `created_at` for output as we are doing for other entities (see https://github.com/gravitee-io/gravitee-management-rest-api/blob/master/gravitee-rest-api-model/src/main/java/io/gravitee/rest/api/model/api/ApiEntity.java#L102).

File: gravitee-management-api-idp/gravitee-management-api-idp-api/src/main/java/io/gravitee/management/idp/api/authentication/UserDetails.java [hunk 2]
```diff
@@ -39,7 +40,20 @@
...
 
+    /**
+     * The user creation date
+     */
+    private Date updatedAt;
```
Comment (brasseld) [inline gravitee-management-api-idp/gravitee-management-api-idp-api/src/main/java/io/gravitee/management/idp/api/authentication/UserDetails.java:51]: Obviously, same here...

File: gravitee-management-api-idp/gravitee-management-api-idp-api/src/main/java/io/gravitee/management/idp/api/authentication/UserDetails.java [hunk 3]
```diff
@@ -39,7 +40,20 @@
...
+
+    /**
+     * The user last connection date
+     */
+    private Date lastConnectionAt;
```
Comment (brasseld) [inline gravitee-management-api-idp/gravitee-management-api-idp-api/src/main/java/io/gravitee/management/idp/api/authentication/UserDetails.java:56]: And finally :)

File: gravitee-management-api-rest/src/test/java/io/gravitee/management/rest/resource/CurrentUserResourceTest.java [hunk 4]
```diff
@@ -67,6 +69,9 @@ public void shouldBeAbleToGetCurrentUser() {
 
         assertThat(response).isNotNull();
         assertThat(response.getStatus()).isEqualTo(HttpStatusCode.OK_200);
+        assertThat(response.readEntity(HashMap.class))
+                .isNotNull()
+                .containsKeys("createdAt", "updatedAt", "lastConnectionAt");
```
Comment (brasseld) [inline gravitee-management-api-rest/src/test/java/io/gravitee/management/rest/resource/CurrentUserResourceTest.java:74]: You should certainly change this also :)

**Review by brasseld — CHANGES_REQUESTED** [review]

**Review by phiz71 — APPROVED** [review]

**Pull request merged**